PhotoStudioPlayer/ViewController.swift

```diff
@@ -141,4 +141,22 @@ class ViewController: NSViewController, AVCaptureVideoDataOutputSampleBufferDele
     @objc private func closeWindow(_ sender: Any) {
         view.window?.close()
     }
+
+    @objc private func switchToCoolStage(_ sender: AnyObject?) {
+        let filter = ChromaKeyFilter.filter(0.15, green: 0.48, blue: 1, threshold: 0.4)
+        sampleBufferChromaKeyFilter = filter
+        previewLayer?.filters = [filter]
```

Comment:
better to avoid reusing filter in two places, i.e. two rendering paths.
try create same filter twice and do like this:

```
sampleBufferChromaKeyFilter = filter1
previewLayer?.filters = [filter2]
```

where `filter1 !== filter2`